mirrord/cli/src/teams.rs

```diff
@@ -1,6 +1,7 @@
 use tokio::process::Command;
 
-const MIRRORD_FOR_TEAMS_URL: &str = "https://mirrord.dev/docs/overview/teams/";
+const MIRRORD_FOR_TEAMS_URL: &str =
```

Comment:
We could make this const local to the function that uses it so that we don't use the same utm_source in other links by mistake.